README.md

```diff
@@ -43,4 +44,13 @@ Now, depending on the storage you want to use, follow the instructions below.
 
 > In case you encounter any problems with finding your AWS credentials please read [this page][1] as a reference.
 
+## Google Cloud Storage
+
+| Environment variable                | Value                                                               |
+|-------------------------------------|---------------------------------------------------------------------|
+| `STORAGES_BACKEND`                  | `storages.backends.google_cloud.GoogleCloudStorage`                 |
+| `STORAGES_GOOGLE_CLOUD_CREDENTIALS` | Your Google Cloud Storage credentials in JSON encoded using Base64. |
+| `STORAGES_BUCKET_NAME`              | The bucket name that you want to use.                               |
```

Comment:
The `STORAGES_BUCKET_NAME` environment variable is already being used by the **AWS S3 storage**. I'm thinking: if we want to be able to define multiple storages simultaneously then we need to make distinctive env var names for the **bucket name** for both: Google Storage and AWS S3 🤔

Reply:
Good point. I wasn't sure if we want to give the option to use both at the same time so it's why I just used one `bucket_name` variable for both. I will change this :)